packages/tx/src/capabilities/eip2718.ts

```diff
@@ -0,0 +1,26 @@
+import { RLP } from '@ethereumjs/rlp'
+import { concatBytes } from '@ethereumjs/util'
+import { keccak256 } from 'ethereum-cryptography/keccak.js'
+
+import { txTypeBytes } from '../util.js'
+
+import { errorMsg } from './legacy.js'
+
+import type { EIP2718CompatibleTxInterface, TypedTransaction } from '../types'
+import type { Input } from '@ethereumjs/rlp'
+
+export function getHashedMessageToSign(tx: EIP2718CompatibleTxInterface): Uint8Array {
+  return keccak256(tx.getMessageToSign())
+}
+
+export function serialize(tx: EIP2718CompatibleTxInterface, base?: Input): Uint8Array {
```

Comment:
Maybe we drop the `Interface` from these names to get it a bit shorter? 🤔 No super strong preference, but we also mostly (exception: these very important reused interfaces in Common) do not explicitly name interfaces with `Interface` or `I`or the like in the monorepo.

Reply:
(but the shortening would be the greater argument here, especially if there are more arguments to follow, then this likely often moves over unnecessarily into two lines)

Reply:
Shortened by removing the unnecessary Interface suffix.